pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -500,6 +510,34 @@ func (gce *GCECloud) ensureTargetPoolAndHealthCheck(tpExists, tpNeedsRecreation
 			glog.Infof("ensureTargetPoolAndHealthCheck(%s): Updated target pool (with %d hosts).", lbRefStr, len(hosts)-maxTargetPoolCreateInstances)
 		}
 	} else if tpExists {
+
+		if hcToDelete != nil {
+			if hc, err := gce.GetHttpHealthCheck(hcToDelete.Name); err != nil {
```

Comment:
Can we construct healthcheck link from the hc name directly (avoiding additional get operation)?

Reply:
Done. Get operation removed.